src/ws-date-picker/ws-date-picker.js

```diff
@@ -89,7 +92,7 @@ export class WSDatePicker extends Component {
    */
   onChange([selectedDate], value) {
     this.setState({value});
-    this.element.dispatchEvent(new CustomEvent('change', {detail: selectedDate}));
+    this.element.dispatchEvent(new CustomEvent('change', {detail: {date: selectedDate, value}, bubbles: true}));
     // Propagate if wanted
     if (this.props.onChange) {
       this.props.onChange(selectedDate);
```

Comment:
It looks like the properties of ws-date-picker are not created, could you ? Or at least we should track the lack of documentation in an Issue